### PR TITLE
Edit SDG12 (WoS)

### PR DESCRIPTION
**Pull request by Caroline-A**
# Edit SDG12

## Target 12.1
- Changed `standards`to be plural only when alone, and allowed it as `standard$` when combined with `national OR regional...` #158
- Moved `cleaner production` so that it no longer has to be combined with `sustainable OR green OR eco...`
- Added `eco-tourism`, `eco-building$` and `bio-econom*´ as synonyms for the one-word variant #148 
- Narrowed `lifestyles$ / life-styles$` to be combined in phrases (`"sustainable lifestyle$" OR "sustainable life style$" OR "green lifestyle$" OR "eco lifestyle$" OR "environmental lifestyle$"`) to avoid medical results.

## Target 12.2
- Phrase 1 & Phrase 2 (prev. 5) - simplified the forestry terms (kept `logging` as stand-alone). Moved forest terms (`*forest* OR woodland*`) so that they have to be combined with *sustainable + management/use* terms, rather than alone with *sustainable* terms. Added `mining` as a natural resource use term. Added `"resource us*" OR "usage" OR "practice$"` to the *management/use* terms. #154 . Moved `"consumption" OR "consume$" OR "consumer$"` from phrase 3 to phrase 1&2. 
- Phrase 3 - Moved `"consumption" OR "consume$" OR "consumer$"` from phrase 3 to phrase 1&2 (`"design*" OR "technolog*"` were removed). Moved *ecolabelling* terms so that they have to be combined with natural resources. #154 . Added `eco-label$` as a synonym for the one-word variant.  #148 . 
- Phrase 4 - `land OR soil` moved to be combined with `"resources" OR "minerals"`. A number of "loss" and "rubbish" terms were removed - other targets cover this and it is not part of the interpretation. 

## Target 12.3
- Phrase 1 - added animal feed terms
- Phrase 2 merged with phrase 1 (capacity terms used as action terms). Confirmed that phrase 2 topic approach added no extra results compared to phrases 1 & 3 topic approach, so we do not lose anything doing this. #149 . Added `legume$`

## Target 12.4 
Clarified the interpretation. Most changes from #150 
- Phrase 1 - `life NEAR/3 cycle` changed to `"lifecycle$" OR "life cycle$"`, mainly added noise from cycle life of batteries. Added `"utilisa*" OR "utiliza*"` to the *use terms* and `safety OR safely` to the *sustainable terms*. Standardised the *management* terms between phrase 1 and 2 - phrase 1 is now `"lifecycle$" OR "life cycle$" OR "end of life" 
          OR "management" OR "handling" OR "packaging" OR "labelling" OR "storage" OR "storing" OR "stored"
          OR "disposal" OR "dispose$" OR "transport*" OR "process*" 
          OR "use$" OR "using" OR "utilisa*" OR "utiliza*" OR "production"`. The terms `"cradle to cradle" OR "cradle-to-cradle" OR "lifecycle analysis" OR "life cycle analysis" OR "lifecycle assessment$" OR "life cycle assessment$"`have been added/moved so that they are no longer required to be combined with *sustainable* terms.
- Phrase 2 - Added  `safety OR safely` to the *sustainable terms*. Standardised the *management* terms between phrases 1 and 2 - phrase 2 is now `"lifecycle$" OR "life cycle$" OR "end of life" 
              OR "management" OR "handling" OR "packaging" OR "labelling" OR "storage" OR "storing" OR "stored"
              OR "disposal" OR "dispose$" OR "transport*" OR "process*" 
              OR "incinerat*" OR "combust*" OR "collect*" OR "treatment" OR "sorting" OR "sorted"`. Removed the superfluous full convention names.  The terms `"cradle to cradle" OR "cradle-to-cradle" OR "lifecycle analysis" OR "life cycle analysis" OR "lifecycle assessment$" OR "life cycle assessment$"`have been added/moved to be with the *recycling* terms (no longer required to be combined with *sustainable* terms).
- Phrase 5 - simplified the `NOT` statement to `"lead to"`

## Target 12.5
Relatively large changes to structure - for conversion to python check the whole string, rather than just these notes.
- Phrase 1 - Allowed some waste terms to occur without being limited to *household/industrial* terms (e.g. agrowaste, ewaste).  Added `eliminate` as an action term. *Topic* - Relatively large structural change that changes the focus away from waste generally, to reductions in waste and waste generation - see the string itself for changes.
- Phrase 1 - Added `urban OR cities OR manufacturing OR industr*` as waste type terms. Added `"agrowaste" OR "solid waste$" ` as *waste* terms.
- Phrase 2 - `life NEAR/3 cycle` changed to `"lifecycle$" OR "life cycle$"`, mainly added noise from cycle life of batteries. Added `zero-waste` as a stand-alone term. Added `recycl* OR re cycl*` so that they can be combined with `sustainable OR environmental....`. Added ` "lifecycle$" OR "life cycle$" OR "cradle to cradle" OR "cradle-to-cradle" OR "material footprint" OR "resource footprint"`with the *recycing* terms. Limited some of the more generic terms -`"bioeconom*" OR "production" OR "produce$" OR "product$" OR "consumption" OR "consume$" OR "consumer$" OR "packaging" OR "matter" OR "biomass"` by limiting what they are combined with. Forced `cleaner production` and `resource efficiency` to be used in combination with *waste* terms (#157). Altered one of the NEAR/5 to NEAR/15. 

## Target 12.6
- Phrase 1 - added `eco-building$` as a synonym for the one-word variant #148 
- Phrase 1 - Moved `cleaner production` so that it no longer has to be combined with `sustainable OR green OR eco...`
- Phrase 1 - Added `corporate OR "industrial sector$" OR "commercial sector$" OR "logistics sector$" OR "manufacturing sector$"` to the *companies* terms. Dropped superfluous `"energy efficien*"` term. #151 and #90
- Phrase 2 - Added `corporate OR "industrial sector$" OR "commercial sector$" OR "logistics sector$" OR "manufacturing sector$"` to the *companies* terms. 
- Phrase 2 -  To the *reporting* terms, added `"environmental information disclosure"` as a phrase, and `"disclos*"` as a synonym to "publish"

## Target 12.7
- Clarified the interpretation
- Added `circular` as a *sustainability* term. Changed `responsib*` to `"socially responsible" OR "social responsibilit*"` - #87 
- Added `"green public procurement$" OR "circular public procurement$"` as standalone terms
- Increased the NEAR/5 distance between *procurement* and *sustainability* terms to NEAR/15
- Truncated `tender` to `tender$`

## Target 12.8
**Major** changes here - in the structure, interpretation and terminology. More terms have been added for SCP (from 12.1); the interpretation has been tightened and thus terms for psychology and behaviour are removed; various phrases for education for sustainable development are added as a stand-alone phrase (phrase 1) (#91). These changes have removed some of the noisy results about health (#88). 
- Phrase 2 - added `eco-building$` and `eco-label*` as a synonym for the one-word variant - note that here there was also an error that was fixed (`ecobuildin$`) #148 . `lifestyles` has been added as suggested in #164. 

## Target 12.a
Edited interpretation to focus on capacity, rather than scientific and technological capabilities. Also did some major changes to the capacity terms - now these are the same between both phrases. 
- Phrase 1 - Changes to the *SCP terms*: Moved `cleaner production` so that it no longer has to be combined with `sustainable OR green OR eco...`. Added `eco-tourism`, `eco-building$` and `bio-econom*` as synonyms for the one-word variant #148 .  Narrowed `lifestyles$ / life-styles$` to be combined in phrases (`"sustainable lifestyle$" OR "sustainable life style$" OR "green lifestyle$" OR "eco lifestyle$" OR "environmental lifestyle$"`) to avoid medical results.
- Phrase 1 and 2 - capacity terms now the same in both phrases
- Phrase 2 - Simplified the power terms, added `energy transitions` and `hydropower` (#156 and #92 ). Added *action* terms to the action approach (same as phrase 1). 

## Target 12.b
- Added `survey*` as a *monitoring* term.
- Removed the *impact* terms as too restrictive - interpretation updated to cover monitoring of sustainable tourism. #93 

## Target 12.c
Interpretation altered in *topic* - removed rationalising/reforming so that the string covers subsities/tax + fossil fuels. *Action* retains rationalising/reforming, now more tightly linked to the *tax/subsidy* terms.
- Phrases 1 and 2 - removed rationalising/reforming from *topic* approach (#153 ). Changed NEAR/5 (between subsidies/tax and fossil fuels) to NEAR/15. Added `conventional energy` as a term for fossil fuels.
- Phrase 1 - added `subsidy` in the *subsidy* terms
- Phrase 2 - Added a not phrase to remove noise from "duty" - `NOT TS=("heavy duty" OR "light duty" OR "medium duty")`

### CODE REVIEW

**Comment by EliSeland** [comment]
I have:
- removed variants of combined words, e.g. reuse, re-use, re use > third variant deleted
- added two-word variant of ecotourism and bioeconom* throughout (#148)
- 2.4. action approach: deleted old version of interpretation as new one had been written
- corrected some misspellings throughout (sewege, emphesise)

Comments: 
- 12.2 phrase 3: There is a suggestion in #154 to move some terms from phrase 3 to 1, "Take out generic terms and incorporate into phrase 1 (e.g. (("saving" OR "save$" OR "conserv*" OR "efficien*") NEAR/5 ("use" OR "using" OR "usage" OR "utili*" OR "consumption")) OR (("sustainab*" OR "responsib*" OR "environmental*" OR "ecological*" OR "eco" OR "green") NEAR/5 ("consumption" OR "consume$" OR "consumer$" OR "design*" OR "technolog*")" - this is not done, was it tested?
- 12.2 noted change in phrase 4: land OR soil moved to be combined with "resources" OR "minerals". Is done in topic, but not action, is that intended?
- 12.3: Documentation needs updating to take into account the fact that two phrases have been merged (#149)
- 12.5: Topic - noted change in phrase 1: "Relatively large structural change that changes the focus away from waste generally, to reductions in waste and waste generation - see the string itself for changes." First part is same as corresponding action string – perhaps worth mentioning in documentation? And perhaps the wording shoud be harmonized, so that in both approaches it says it is about "reducing waste and waste generation"?
- 12.6: Comment in (#151) about using the SCP string (suppose it is meant in phrase 2, as it says "(as used in phrase 1)") - if correctly understood, it has not been implemented - has it been tested?
- 12.8: Noted that ; "education for sustainable development is added as a stand-alone term (#91)" - is that so? It looks to me as if it has been removed - I may misunderstand, but perhaps best to check? I also find the documentation of phrase 1 a little unclear - perhaps consider rewording? Also look at grammar of last sentence of documentation for phrase 2. For phrase 2, noted that "eco-label*" was added, but it is "eco-labels" and "eco-labelling" that is added, is that intended/ok?
- 12.a: The interpretation is a bit heavy - I wonder - is it "the promoting capacity of developing countries for sustainable consumption and production" (= as it is now, action approach), or "the capacity of developing countries for promoting sustainable consumption and production" (to me the latter is easier to grasp, but unsure if something goes missing?
- 12.a: noted "capacity terms now the same in both phrases", but  looks like "technological capacity" is in phrase 2, but not 1
- 12.a: not quite sure all the comments/suggestions in (#92) are addressed, a bit hard to follow (terms suggested used alone have been removed), perhaps take an extra check?
- 12.b documentation - says specifying terms were "found from" - is it better to say "found in" or "taken from"?

**Comment by Caroline-A** [comment]
> 12.2 phrase 3: There is a suggestion in https://github.com/SDGforskning/sdg-strings/issues/154 to move some terms from phrase 3 to 1, "Take out generic terms and incorporate into phrase 1 (e.g. (("saving" OR "save$" OR "conserv*" OR "efficien*") NEAR/5 ("use" OR "using" OR "usage" OR "utili*" OR "consumption")) OR (("sustainab*" OR "responsib*" OR "environmental*" OR "ecological*" OR "eco" OR "green") NEAR/5 ("consumption" OR "consume$" OR "consumer$" OR "design*" OR "technolog*")" - this is not done, was it tested?

The first part is retained in phrase 3 (it was better there after phrase 1 has a looser connection to natural resources, whereas phrase 3 is closer). The second part (`"consumption" OR "consume$" OR "consumer$"`) was moved to phrase 1 (design and technology were removed - have added a comment on this now).
